source/api/content-state/0.1/index.md

```diff
@@ -0,0 +1,289 @@
+
+A description like this can be expressed in purely IIIF terms, as an annotation that targets the intended part of the resource. This _content state_ is distinct from the state of a particular viewer's user interface. A viewer state is likely to be client-specific and would concern which panels are open, which options are selected and similar user interface details.  
+
+### 2.1. Annotation Model for Content State
+
+The target of the annotation is the IIIF resource, using exactly the same patterns as any other IIIF-targeting annotation that a IIIF viewer might encounter. A content state annotation has the motivation `highlighting`.
+
+The target could be any resource described by the Presentation API, for example, a:
+
+* Manifest
+* Range
+* Canvas
+* spatial or temporal fragment of a Canvas
+* spatial or temporal point on a Canvas
+
+The annotation must contain enough information about dereferenceable resources to show the content in context. For example, a Canvas is almost certainly not enough information for a viewer to show the intended view; the Manifest the canvas is part of needs to be indicated so that the client can load it first and then find the Canvas within it.
+
+### 2.2 Form of annotation
+
+The content state data may be passed to a client in one of several forms, and a client _SHOULD_ be able to accept and process it in all of these forms. 
+
+* As the URL of an annotation, that the client must de-reference
+* As JSON-LD, in the form of a full annotation
+* As JSON-LD, as the value of the `target` property of an implied annotation with the motivation `highlighting`
+
+If a client is capable of reading the content state from the value of an HTTP GET or POST request parameter, it _MUST_ look for the content state in a request parameter called `iiif-content`.
+
+If a client is capable of reading the content state from an HTML attribute, the client _SHOULD_ look for the content state in an attribute called `data-iiif-content`.
+
+Clients _MAY_ implement other mechanisms of accepting content state data; some suggestions are given in the next section.
+
+A client _MUST_ be able to read and process the value in any of the three forms listed above. Consider the following, simplest possible content state annotation:
+
+```json
+{
+  "@context": "http://iiif.io/api/presentation/{{ page.major }}/context.json",
+  "id": "https://example.org/annotation-server/bookmarks/b1",
+  "type": "Annotation",
+  "motivation": ["highlighting"],
+  "target": {
+     "id": "https://example.org/iiif/item1/manifest",
+     "type": "Manifest"
+  }
+}
+```
+
+If the string `https://example.org/annotation-server/bookmarks/b1` is passed as the value of `iiif-content` or `data-iiif-content`, the client _MUST_ load the resource at that URL and process it. The resource at that URL _MUST_ be the full annotation as in the example above. 
```

Comment:
This seems more like protocol and less like Form of Annotation? (and below)

Reply:
Yeah, this section has lots of protocol rather than form in it. Needs rewriting - for v0.3? To get this out there...